name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
yarn.client.failover-retries-on-socket-timeouts,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
mapreduce.job.running.map.limit,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701312551309_0300_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.client.failover-retries-on-socket-timeouts,0,"ERROR	Task: attempt_1701312551309_0300_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.router.pipeline.cache-max-size,25,"ERROR	Task: attempt_1701312551309_0300_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.job.running.map.limit,0,"ERROR	Task: attempt_1701312551309_0300_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701312551309_0300_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.client.failover-retries-on-socket-timeouts,0,"INFO	Diagnostics report from attempt_1701312551309_0300_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.router.pipeline.cache-max-size,25,"INFO	Diagnostics report from attempt_1701312551309_0300_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
mapreduce.job.running.map.limit,0,"INFO	Diagnostics report from attempt_1701312551309_0300_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
